scripts/release/relnotes.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import requests
+import subprocess
+
+GCS_BASE_URL = "https://www.googleapis.com/storage/v1/b/bazel/o?delimiter=/"
+
+
+def _paginated_get(base_url):
+  """GETs a URL from GCS, respecting the pagination token if any."""
+  next_page_token = None
+  while True:
+    url = base_url
+    if next_page_token is not None:
+      url += f"&pageToken={next_page_token}"
+    r = requests.get(url)
+    r.raise_for_status()
+    json = r.json()
+    yield json
+    if "nextPageToken" in json:
+      next_page_token = json["nextPageToken"]
+    else:
+      break
+
+
+def _version_sort_key(v):
+  # Converts "5.3.2" into [5, 3, 2] so that we get a natural lexicographical
+  # sort. Note that we have a version "0.2.2b" so we just convert any non-digit
+  # segment into 9999.
+  return [int(s) if s.isdigit() else 9999 for s in v.split(".")]
+
+
+def get_latest_release():
+  """Discovers the latest stable release name from GCS."""
```

Comment:
If you want to simplify this, you can use 
```
#!/bin/env python3

from urllib import response
import requests
response = requests.get('https://github.com/bazelbuild/bazel/releases/latest')
print(response.url)
print(response.url.split("/")[-1])
```
output is
```
https://github.com/bazelbuild/bazel/releases/tag/5.3.2
5.3.2
```

Reply:
We are using similar approach at https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/setup-windows.ps1#L185-L191

Reply:
wow, that's so much simpler :) done.